security/encrypt-cluster-pod-traffic.md

```diff
@@ -57,7 +57,9 @@ To install WireGuard on the default Amazon Machine Image (AMI):
 <%
 To install WireGuard for OpenShift v4.3:
```

Comment:
Are these instructions only true for 4.3? On Enterprise at least, we only support OCP 4.6 and newer.

Reply:
I tested this against 4.6 cluster. Do we have a matrix of the version support?